hw5_zadacha6.py

```diff
@@ -0,0 +1,15 @@
+"""
+Вводится число найти его максимальный делитель,
+являющийся степенью двойки. 10(2) 16(16), 12(4).
+"""
+user = int(input("Введите число, для определения максимального делителя "))
+
+
+def get_maxdiv(num):

```

Comment:
it doesn't solve task - for 10 it should be 2, not 8

Reply:
fixed